subt/docker/robotika/entrypoint.bash

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+source /opt/ros/melodic/setup.bash
```

Comment:
This is not needed. See for example https://answers.ros.org/question/292566/what-is-the-difference-between-local_setupbash-and-setupbash/?answer=292610#post-id-292610 for the difference between `setup.bash` and `local_setup.bash` - `setup.bash` initializes also all parent workspaces.

Reply:
If I join the solution docker without this, I am not able to call standard ROS commands from the commandline (eg rostopic). Debugging is very difficult without this.

Reply:
I am not able to run anything from ROS with or without this change when I exec into the running container. I believe the way to deal with this problem is to instead append `source ~/subt_solution/devel/setup.sh` to `.bashrc`. That is what I am doing here

https://github.com/robotika/osgar/pull/453/files#diff-71523998535d95fa0c75c65bb6a2818dR7-R8